bin/sync_all

```diff
@@ -8,7 +8,7 @@ fi
 ./manage.py migrate --noinput --database bedrock
 ./manage.py rnasync
 ./manage.py cron update_ical_feeds
```

Comment:
We might, or we might not. We'll have to test. The router should make most things do the right thing. The p-d one was only a problem because of the manual transaction management it does.